user/src/com/google/gwt/junit/JUnitShell.java

```diff
@@ -1092,7 +1091,8 @@ void compileForWebMode(ModuleDef module, Set<String> userAgents)
     try {
       success = Compiler.compile(getTopLogger(), options, module);
     } catch (Exception e) {
-      getTopLogger().log(Type.ERROR, "Compiler aborted with an exception ", e);
+      // noinspection ThrowableNotThrown
+      CompilationProblemReporter.logAndTranslateException(getTopLogger(), e);
```

Comment:
Shouldn't this throw the returned exception?
```suggestion
      throw CompilationProblemReporter.logAndTranslateException(getTopLogger(), e);
```

Reply:
It very reasonably could, but see one line (logical) line down, an exception will be thrown regardless of whether the compiler returned false, or an exception was thrown and logged.

My goal was to only change the logging rather than the rest of the behavior here.